packages/local-environment/README.md

```diff
@@ -0,0 +1,34 @@
+## Bootup instructions
+
```

Comment:
Mention these commands should be run from monorepo root

Reply:
Done